### PR TITLE
Update cli readme

### PR DESCRIPTION
**Pull request by janewang**
### What

Update readme and contributing.md

### Why

Provide guidance on the cli and how to contribute

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,48 @@
...
+# Soroban CLI (soroban-cli)
 
 This repo is home to the [Soroban CLI](https://github.com/stellar/soroban-tools/tree/main/cmd/soroban-cli): The command-line multi-tool for running and deploying Soroban contracts.
 
-Soroban: https://soroban.stellar.org
 
-# Adding git hooks
+## Install the CLI
+Install [the latest soroban-cli version](/releases).
+`cargo install --locked --version 20.3.1 soroban-cli`
```
[the remark this answers is not in the document]
Comment (willemneal) [reply README.md:8]: I would also add that you can install with `cargo-binstall` so after `cargo install cargo-binstall`

```
cargo binstall -y soroban-cli
```
This downloads the binary from github and skips needing to compile, which is very useful in CI.

**Review by willemneal — APPROVED** [review]
Looks good! Just one nit.

**Comment by willemneal** [comment]
It looks like a new example was added. So you need to run `cargo md-gen` to generate the markdown docs.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (janewang) [reply README.md]: Thanks added

**Pull request merged**